src/app/(main)/my-voyage/[teamId]/sprints/sprintsService.ts

```diff
@@ -141,17 +142,13 @@ export interface AddSectionProps extends AddSectionType {}
 export interface EditSectionProps extends EditSectionType, EditSectionBody {}
 
 export interface FetchSprintsResponse extends SprintsResponse {
-  voyage: {
+  sprints: {
     id: number;
-    number: string;
-    sprints: {
-      id: number;
-      number: number;
-      startDate: string;
-      endDate: string;
-      teamMeetings: { id: number }[];
-    }[];
-  };
+    number: number;
+    startDate: string;
+    endDate: string;
+    teamMeetings: { id: number }[];
+  }[];
 }
```

Comment:
So I'm confused why you have this. 

1. this interface isn't really doing anything because you already have this shape of data in the sprints response interface. 
2. I checked where you're using the fetchsprintsreponse and you're using it in the fetchSprints function, which you set the return type as fetchsprintsreponse, but when I inspected the data, it's just the sprints response.

I tried removing this fetchsprintsresponse and updating the fetchsprint function type and it works, so this interface seems unnecessary.

Reply:
Yeah... but I want to keep `FetchSprintsResponse  ` for consistency, can I do it like this:
```
export interface FetchSprintsResponse extends SprintsResponse {}
```

Reply:
> Yeah... but I want to keep `FetchSprintsResponse ` for consistency, can I do it like this:
> 
> ```
> export interface FetchSprintsResponse extends SprintsResponse {}
> ```

Yup, if you just wanted it for sake of naming, then that would be better. There's no need to add the other stuff since it's already a part of the sprints response interface